fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -229,9 +232,10 @@ private boolean attendedOrOrganized(Lao lao, RollCall rollcall) {
     String pk;
     try {
       pk =
-          Base64.getUrlEncoder()
-              .encodeToString(
-                  Wallet.getInstance().findKeyPair(firstLaoId, rollcall.getPersistentId()).second);
+          wallet
```

Comment:
please rename pk so that we don't need to dig into the code to find if its public or private

Reply:
Done !